plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/TrinoS3ClientFactory.java

```diff
@@ -104,6 +106,12 @@ private AmazonS3 createS3Client(Configuration config)
                 .withUserAgentPrefix(userAgentPrefix)
                 .withUserAgentSuffix(enabled ? "Trino-select" : "Trino");
 
+        String connectTtlValue = config.get(S3_CONNECT_TTL);
+        if (nonNull(connectTtlValue)) {
```

Comment:
```suggestion
        if (!isNullOrEmpty(connectTtlValue)) {
```

Reply:
> 

@findinpath Thank you for your advice, i have trimmed the width of the commit message to 80, modified the explanation in the .rst doc, and replaced the function that judges whether the string type is null.  

Please review again~